services/QuillLMS/client/app/bundles/Staff/components/comprehension/rulesAnalysis/rulesAnalysis.tsx

```diff
@@ -29,6 +29,18 @@ const apiOrderLookup = {
   'rules-based-3': 8,
 }
 
+function strongTextClassName(percentage) {
+  if (percentage >= 90) { return 'green' }
+  if (percentage >= 60) { return 'orange' }
+  return 'red'
+}
+
+function weakTextClassName(percentage) {
+  if (percentage <= 10) { return 'green' }
+  if (percentage <= 40) { return 'orange' }
+  return 'red'
```

Comment:
might be worth turning these into consts and pulling them out into the helper function file? I could see this potentially being used in other areas